my-account/app/myaccount/login/SignUpFormData.java

```diff
@@ -0,0 +1,88 @@
+package myaccount.login;
+
+import io.sphere.sdk.models.Base;
+import play.data.validation.Constraints;
+
+public class SignUpFormData extends Base {
+    @Constraints.Required
```

Comment:
everybody needs a title to sign up?

I think the form is screwed up with salutation.